src/meta/src/barrier/mod.rs

```diff
@@ -679,14 +712,14 @@ where
     }
 
     /// Resolve actor information from cluster and fragment manager.
```

Comment:
Ditto for docs. Better to explain what `changed_table_id` is and why we need that.